src/sentry/static/sentry/app/views/settings/account/accountNotificationFineTuning.jsx

```diff
@@ -62,6 +62,8 @@ const ACCOUNT_NOTIFICATION_FIELDS = {
       "Reports contain a summary of what's happened within the organization."
     ),
     type: 'select',
+    // API only saves organizations that have this disabled, so we should default to "On"
+    defaultValue: 1,
```

Comment:
Do we need to update the select component to accept numbers as default values? (currently only accepts a string)

Reply:
Updated https://github.com/getsentry/sentry/pull/11505/commits/d04f17cd6d87fb5797ed63a2e091a95efcc59f80